aws/blobstore.tf

```diff
@@ -0,0 +1,8 @@
+resource "aws_s3_bucket" "veto_crls" {
```

Comment:
Would you mind changing this file to aws/s3.tf

Reply:
Fixed with d731be7